weis/aeroelasticse/openmdao_openfast.py

```diff
@@ -2101,9 +2076,6 @@ def get_monopile_loading(self, sum_stats, extreme_table, inputs, outputs):
         outputs['monopile_maxMy_My'] = spline_My(z)
         outputs['monopile_maxMy_Mz'] = spline_Mz(z)
 
-        for k in ['Fx','Fy','Fz','Mx','My','Mz']:
-            outputs[f'tower_monopile_maxMy_{k}'] = np.r_[outputs[f'monopile_maxMy_{k}'], outputs[f'tower_maxMy_{k}']]
-
```

Comment:
Does this mean that the loading along the tower is represented by `outputs['monopile_maxMy_*']` ?

Reply:
No, there is a separate `outputs['tower_maxMy_*']` in the function just above the one you called out:
https://github.com/WISDEM/WEIS/blob/jacketse/weis/aeroelasticse/openmdao_openfast.py#L2009-L2014